zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala

```diff
@@ -149,6 +149,23 @@ class InMemorySpanStore extends SpanStore with CollectAnnotationQueries {
       .toList
   }
 
+  override protected def getTraceIdsByDuration(
+    serviceName: String,
```

Comment:
@adriancole 

Q1 - if I am reading sql query correctly, the duration limit applies to individual span duration, not the overall trace duration, is that correct?

Q2 - what are the semantics of serviceName here? Any service involved in a span satisfying the duration limit? What about services involved in a long trace, but not in the long span in that trace?

Q3 - shouldn't we include span name in the search?